backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/importer/AcceptingStoresImporter.kt

```diff
@@ -0,0 +1,76 @@
+package xyz.elitese.ehrenamtskarte.importer
+
+import com.beust.klaxon.Klaxon
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.postgis.Point
+import xyz.elitese.ehrenamtskarte.database.*
+import xyz.elitese.ehrenamtskarte.importer.annotations.BooleanField
+import xyz.elitese.ehrenamtskarte.importer.annotations.DoubleField
+import xyz.elitese.ehrenamtskarte.importer.annotations.IntegerField
+import xyz.elitese.ehrenamtskarte.importer.converters.booleanConverter
+import xyz.elitese.ehrenamtskarte.importer.converters.doubleConverter
+import xyz.elitese.ehrenamtskarte.importer.converters.integerConverter
+import xyz.elitese.ehrenamtskarte.importer.dataqa.FreinetDataFilter
+import xyz.elitese.ehrenamtskarte.importer.types.Category
+import xyz.elitese.ehrenamtskarte.importer.types.FreinetAcceptingStore
+import xyz.elitese.ehrenamtskarte.importer.types.FreinetData
+
+object AcceptingStoresImporter {
+
+    fun importFromJsonFile(jsonContent: String) {
+        val freinetData = parseFreinetJson(jsonContent)!!
+
+        val categoriesJson = AcceptingStoresImporter::class.java
+                .getResource("/freinet_import/categories.json").readText()
+        val categories = Klaxon().parseArray<Category>(categoriesJson)!!
+        transaction {
+            categories.forEach { CategoryEntity.new(it.id) { name = it.name } }
+        }
+        println("Filtering out invalid accepting stores, count before filtering: ${freinetData.data.size}")
+        val filteredFreinetAcceptingStores = FreinetDataFilter.filterOutInvalidEntries(freinetData.data)
+        println("Count after filtering: ${filteredFreinetAcceptingStores.size}")
+        importAcceptingStores(filteredFreinetAcceptingStores)
+    }
+
+    private fun parseFreinetJson(freinetJson: String) = Klaxon()
+            .fieldConverter(IntegerField::class, integerConverter)
+            .fieldConverter(DoubleField::class, doubleConverter)
+            .fieldConverter(BooleanField::class, booleanConverter)
+            .parse<FreinetData>(freinetJson)
+
+    private fun importAcceptingStores(acceptingStores: List<FreinetAcceptingStore>) {
+        println("Inserting data into db")
+        try {
+            transaction {
+                for (acceptingStore in acceptingStores) {
+                    val address = AddressEntity.new {
+                        street = acceptingStore.street
+                        houseNumber = "10"
+                        postalCode = acceptingStore.postalCode
+                        locaction = acceptingStore.location
+                        state = "de"
```

Comment:
Is state fine here or should it be "Germany"? We should define what should be in this field.

E.g. Alpha-2 codes: https://www.iban.com/country-codes